src/main.cpp

```diff
@@ -6124,7 +6132,7 @@ class CMainCleanup
     #ifdef _DEBUG
                 nUpdatePeriod = 100,
     #else
-                nUpdatePeriod = 3000, //10000,   // pure guess for a decent update period ~1 second
+                nUpdatePeriod = 300, //3000, //10000,   // pure guess for a decent update period ~1 second
     #endif
                 nEstimate;
 
```

Comment:
```c++
main.cpp
    while (true)
    {
-        // Don't bother if send buffer is too full to respond anyway
-        if (pfrom->vSend.size() >= SendBufferSize())
-            break;
...
+        if ( 0 != pfrom->vSend.size() )
+        {        // Don't bother if send buffer is too full to respond anyway
+            if (pfrom->vSend.size() < SendBufferSize())
+          //if (pfrom->vSend.size() >= SendBufferSize())
+          //      break;
+          //else
+            {    // Process message
+                bool 
+                    fRet = false;
```
=> Why don't we keep the old logic here? If the send buffer is too full, we will break the while loop immediately, no need to take time to do other things such as serialize and checksum